.github/workflows/release.yml

```
@@ -10,46 +10,28 @@ env:
       - name: Install dependencies
-        run: pnpm install --frozen-lockfile
+        run: bun install --frozen-lockfile
 
       - name: Build
         id: build
-        run: pnpm build
+        run: bun run build
 
       - name: Create Release
         id: create_release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)